clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -2103,31 +2097,36 @@ void handleFailure(Throwable throwable) {
 
     @Override
     public AlterConfigsResult incrementalAlterConfigs(Map<ConfigResource, Collection<AlterConfigOp>> configs,
-                                                                 final AlterConfigsOptions options) {
+                                                      final AlterConfigsOptions options) {
```

Comment:
The code in `incrementalAlterConfigs` is almost identical to the code in `alterConfigs`. I wonder if we could share more of the logic between the two. Have you tried already?